README.md

```diff
@@ -1 +1,40 @@
-# TodoList
+# TodoList (JavaScript)
+
+> This is a small Todo list project, it has some OOP elements and a small activity list for every project.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo
```

Comment:
Your live link seems broken, kindly look into it.

The link is not working at my end.

Reply:
The page was updated and relaunched it should work now.

I'm way too tired of this project to change the UI, I'm not a UI/UX designer, maybe in the future I'll do it as a learning experience on UI/UX for now all that matters to me is the functionality.